javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+const width = 5;
+const height = 4;
+const area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+const bill = 10;
+const payment = 15;
+const change = bill - payment;
```

Comment:
This should be swapped! You should pay more than the bill, otherwise you won't cover your tab 😉